rules/include-annual-cost/rule.md

```diff
@@ -0,0 +1,35 @@
+---
+type: rule
+title: Do you include annual cost?
```

Comment:
@AttackOnMorty 

I think the title should say "Do you include the annual cost in quotes?"

ATM it's a little vague